doc/nrf/protocols/zigbee/qsg.rst

```diff
@@ -102,6 +102,10 @@ For this quick start guide, we will install the following software:
 * Microsoft's |VSC| - The recommended IDE for the |NCS|.
 * |nRFVSC| - An add-on for |VSC| that allows you to develop applications for the |NCS|.
 * nRF Command Line Tools - A set of mandatory tools for working with the |NCS|.
```

Comment:
These tools aren't actually used in the quick start guide, and seem to be left over from a previous version. Okay to just remove this (and Line 101 while we're at it), so the deprecation note isn't needed.

Reply:
All three lines, @wiba-nordic , or just line 104?

Reply:
Just 104. VSC is used in the steps. Sorry, wasn't precise in my initial comment.